values-schema.yaml

```diff
@@ -1335,6 +1540,27 @@ properties:
             type: string
           schedule:
             type: string
+      vault:
+        type: object
+        additionalProperties: false
+        properties:
+          enabled:
+            type: boolean
+            default: true
+          seal:
+            description: Use Cloud KMS to encrypt and decrypt the master key
+            oneOf:
+              - title: default
+                $ref: '#/definitions/vaultDefaultSeal'
+              - title: gcpckms
+                $ref: '#/definitions/vaultGoogleSeal'
+              - title: awskms
+                $ref: '#/definitions/vaultAwsSeal'
+              - title: azurekeyvault
+                $ref: '#/definitions/vaultAzureSeal'
```

Comment:
I believe the above props should go here again.

Reply:
Noooooo, otherwise I am totally lost with this complex schema and not able to easily maintain it.

Reply:
First we need to solve comment in: .demo/env/clouds/aws/demo/overrides.aws-demo.yaml

Reply:
done